bigquery_etl/cli/metadata.py

```diff
@@ -111,23 +119,41 @@ def update(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None
     ConfigLoader.get("default", "project", fallback="moz-fx-data-shared-prod")
 )
 @sql_dir_option
-def publish(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None:
+@parallelism_option()
+def publish(
+    name: str, sql_dir: Optional[str], project_id: Optional[str], parallelism: int
+) -> None:
     """Publish Bigquery metadata."""
-    client = bigquery.Client(project_id)
-
     table_metadata_files = paths_matching_name_pattern(
         name, sql_dir, project_id=project_id, files=["metadata.yaml"]
     )
 
-    for metadata_file in table_metadata_files:
-        project, dataset, table = extract_from_query_path(metadata_file)
-        try:
-            metadata = Metadata.from_file(metadata_file)
-            publish_metadata(client, project, dataset, table, metadata)
-        except FileNotFoundError:
-            print("No metadata file for: {}.{}.{}".format(project, dataset, table))
+    if parallelism > 0:
```

Comment:
This is needed to make the unit tests execute correctly. Mocked objects and multiple processes don't seem to work. This is similar to what we have for the query CLI commands: https://github.com/mozilla/bigquery-etl/blob/cfdc045ff5a2914bfcbb9ad5e2b4a78cc4407a1d/bigquery_etl/cli/query.py#L776